pyro/infer/mcmc/nuts.py

```diff
@@ -110,17 +120,12 @@ def __init__(self,
         #     + The probability of the states becoming negligible: p(z, r) << u,
         # here u is the "slice" variable introduced at the `self.sample(...)` method.
         # Denote E_p = -log p(z, r), E_u = -log u, the third condition is equivalent to
-        #     sliced_energy := E_p - E_u >= some constant =: max_sliced_energy.
+        #     sliced_energy := E_p - E_u > some constant =: max_sliced_energy.
         # This also suggests the notion "diverging" in the implemenation:
         #     when the energy E_p diverges from E_u too much, we stop doubling.
         # Here, as suggested in [1], we set dE_max = 1000.
         self._max_sliced_energy = 1000
 
-        # Set a flag to decide if we want to eliminate the initial point from the candidates to
```

Comment:
In Stan, multinomial sampling version of NUTS includes initial point while slice sampling ignores initial point. To compromise, I remove this flag to avoid confusion. (a developer of Stan said that "including the original point is important for avoiding periodicity.")

Reply:
Does this cause any issues with slice sampling? I am guessing not because that's what we had earlier too.